config/kitty/kitty.conf

```diff
@@ -0,0 +1,63 @@
+# vim: fileencoding=utf-8 ft=conf
+
+# Settings
+close_on_child_death yes
```

Comment:
😱

Reply:
It's a great option if you have most of your desktop in the terminal, actually. :-)

Without this the terminal will not close if it ends by forking another GUI process. Even if it's completely detached, `setsid` and have no connection to the terminal's stdin, stdout or stderr.
It's the default behavior of almost all other terminal emulators, actually.